pkg/packagekit/updates.jsx

```diff
@@ -1484,11 +1483,13 @@ class OsUpdates extends React.Component {
                                         </TextContent>
                                     }
                     />
-                    <CodeBlockCode className='error-log'>
-                        {this.state.errorMessages
-                                .filter((m, index) => index == 0 || m != this.state.errorMessages[index - 1])
-                                .map(m => <span key={m}>{m}</span>)}
-                    </CodeBlockCode>
+                    <CodeBlock>
```

Comment:
You should be able to add `className="pf-u-ml-auto pf-u-mr-auto"` to `CodeBlock` to make it align in the center. (We're already important the spacing helper classes on `updates.scss`, so adding the classes should be enough.)

Reply:
BTW: pf-u-ml-auto stands for "patternfly utility, margin-left: auto`

The utility classes are oddly named if you don't know what they mean, so I thought I should point it out. :grin: 

For reference, here's the spacing page: https://www.patternfly.org/v4/utilities/spacing

Reply:
OH, this can actually be shortened to: `pf-u-mx-auto` for adding margin to the sides! TIL ("Today, I learned...")